picard/ui/mainwindow.py

```diff
@@ -535,6 +538,9 @@ def create_actions(self):
         self.open_folder_action.setEnabled(False)
         self.open_folder_action.triggered.connect(self.open_folder)
 
+        self.check_update_action = QtWidgets.QAction(_("&Check for Update"), self)
+        self.check_update_action.triggered.connect(partial(self.check_for_update, show_always=True))
```

Comment:
Since `show_always=True` by default, no need of `partial` here

Reply:
For some strange reason if I remove the `partial`, a `show_always` value of `False` is being passed to the function.

Reply:
> For some strange reason if I remove the partial, a show_always value of False is being passed to the function.

Nothing strange, i just forgot it was a pyqt slot, and those aren't supposed to have default argument.

Basically you should have a method:

```python
self.check_update_action.triggered.connect(self.do_check_update)

def do_check_update(self):
  self.check_for_update(show_always=True)
```

So, basically, `partial()` does the same, my bad.

Though i think you could make the `check_for_update()` argument mandatory, it is called in 2 places and argument is explicitely set each time.

Reply:
Right.  I should have seen that.  Changes made as suggested, including making the `check_for_update()` argument mandatory.  Thanks.